android/app/src/main/java/app/organicmaps/search/SearchResult.java

```diff
@@ -141,4 +144,27 @@ public Spannable getFormattedTitle(@NonNull Context context)
 
     return builder;
   }
+
+  public Spannable getFormattedAddress(@NonNull Context context)
+  {
+    final String address = description != null ? description.region : null;
+    final SpannableStringBuilder builder = new SpannableStringBuilder(address);
+
+    if (descHighlightRanges != null)
+    {
+      final int size = descHighlightRanges.length / 2;
+      int itr = 0;
+
+      for (int i = 0; i < size; i++)
+      {
+        final int start = descHighlightRanges[itr++];
+        final int len = descHighlightRanges[itr++];
+
+        builder.setSpan(new StyleSpan(Typeface.BOLD), start, start + len, Spanned.SPAN_EXCLUSIVE_EXCLUSIVE);
+      }
+    }
+
+    return builder;
+  }
```

Comment:
Can avoid copy-paste in getFormattedTitle and getFormattedAddress with adding helper fucntion.